pkg/doc/verifiable/credential.go

```diff
@@ -674,9 +674,11 @@ func (vc *Credential) JWTClaims(minimizeVC bool) (*JWTCredClaims, error) {
 	return newJWTCredClaims(vc, minimizeVC)
 }
 
-// SubjectID gets ID of single subject if present or
+// subjectID gets ID of single subject if present or
 // returns error if there are several subjects or one without ID defined
-func (vc *Credential) SubjectID() (string, error) {
+// subjectID gets ID of single subject if present or
+// returns error if there are several subjects or one without ID defined
+func subjectID(subject interface{}) (string, error) {
```

Comment:
I found the issue with `subjectID()` as it did not support custom structures case. So I had to fix this in this PR to enable the VC encoding example with custom `UniversityCredentialSubject` subject.